.github/workflows/unit-test-pr.yml

```diff
@@ -20,4 +19,4 @@ jobs:
       - name: Install and Build 🔧
         run: |
           pnpm install
-          pnpm run --dir app test:unit
+          pnpm run test:unit
```

Comment:
[praise](https://conventionalcomments.org/#labels): thanks for not forgetting anything, even the CI :)